.github/workflows/dev.yaml

```
@@ -59,16 +74,26 @@ jobs:
         with:
           context: .
           push: ${{ github.event_name != 'pull_request' }}
           file: ./cmd/seaweedfs-csi-driver/Dockerfile
           platforms: linux/amd64, linux/arm, linux/arm64, linux/386
-          tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
+          tags: ${{ steps.docker_meta_csi.outputs.tags }}
+          labels: ${{ steps.docker_meta_csi.outputs.labels }}
+
+      - name: Build mount-service image
+        uses: docker/build-push-action@v2
```

actionlint (1.7.9)

the runner of "docker/build-push-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)